docs/requirements.txt

```diff
@@ -1,3 +1,3 @@
-sphinx >= 1.6.5, < 2
+sphinx >= 1.6.5, < 1.7.3
```

Comment:
It helps to also write **why** it breaks in the commit message. :wink:

Reply:
Well, I don't know the reason :stuck_out_tongue_closed_eyes:

Reply:
In this case you can write "I don't know why". :stuck_out_tongue: At least we know we don't know. :sweat_smile: